packages/server/lib/browsers/utils.ts

```diff
@@ -157,6 +158,27 @@ async function executeBeforeBrowserLaunch (browser, launchOptions: typeof defaul
   return launchOptions
 }
 
+interface AfterBrowserLaunchDetails {
+  webSocketDebuggerUrl: string | never
+}
+
+async function executeAfterBrowserLaunch (browser: Browser, options: AfterBrowserLaunchDetails) {
+  if (plugins.has('after:browser:launch')) {
```

Comment:
iirc, with chrome if we crash, we will re-launch the browser. does this re-trigger if that happens?

Reply:
Do we re-launch the browser? From what I can tell, we detect the crash and report it, which seems like the appropriate behavior.